server/src/test/java/io/unitycatalog/server/utils/OptionParserTest.java

```diff
@@ -0,0 +1,15 @@
+package io.unitycatalog.server.utils;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class OptionParserTest {
+
+  @Test
+  void testParseCLIOptions() {
+    OptionParser optionParser = new OptionParser();
+    optionParser.parse(new String[] {"-p", "8081"});
+    assertEquals(8081, optionParser.getPort());
```

Comment:
can we also test the help and version options?

Reply:
Addressed